server/api_server/sessions_endpoint.py

```
@@ -349,11 +351,11 @@ def get_session_events(event_type, period_date):
         if event_type in ("sessions", "missing"):
             # Duration
             if row[5] and row[6]:
-                delta = format_date_diff(row[5], row[6])
+                delta = format_date_diff(row[5], row[6], tz_name)
```

Ruff (0.14.3)

Undefined name `tz_name`

(F821)